composer.json

```diff
@@ -36,7 +36,8 @@
     }
   },
   "require": {
-    "alek13/slack": "^2.0"
+    "alek13/slack": "^2.0",
+    "google/apiclient": "^2.0"
```

Comment:
Still sorting an issue with auto loading. That breaks the test instance.

Reply:
This turned out an issue with test instance deployments. For now we worked around it by merging the package first.